apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -27,7 +27,7 @@ Page {
     function removeProtocolTypeFromUri(uri) {
         if (uri.length === 0)
             return uri
-       return uri.replace(/(^\w+:|^)\/\//, '')
+        return uri.replace(/(^\w+:|^)\/\//, '')
     }
 
     SilicaFlickable {
```

Comment:
I guess it was hidden before because the page contents was shorter than the screen, but it's common practice to add some space to the bottom of the content so that it doesn't touch the bottom of the screen when scrolled. I.e. add something like this into the column around line 40:
```
        bottomPadding: Theme.paddingLarge
```

Reply:
Fixed :)
Added the padding and removed all references to AccessPolicy.browserEnabled

Reply:
Should you drop `import Sailfish.Policy 1.0` as well?

Reply:
All of these look addressed now, so I'll resolve this conversation for clarity.